Update packaging scripts

jhsimpson opened the pull request:
updating debian package scripts, to fix issue 7916, as well as update version number to 0.6.0

jhsimpson committed:
updating debian packaging for 0.6.0 release

refs #7684  preparing for 0.6.0 release.
add new log directory and update version number.

jhsimpson committed:
include new static files on upgrade

refs #7916

Change postinst script to merge in new templates and static files
when upgrading.

mistydemeo commented on debian/postinst:
```diff
@@ -31,12 +32,12 @@ echo "configuring django database and static files"
 chown -R archivematica:archivematica /var/archivematica/storage_service
 chown -R archivematica:archivematica /var/archivematica/.storage-service
 chown -R archivematica:archivematica /usr/share/python/archivematica-storage-service
+chown -R archivematica:archivematica /var/log/archivematica
```
Do we want to pre-emptively set this to `g+ws` to prepare for artefactual/archivematica#34?

jhsimpson replied:
I'm not sure.  There is no guarantee that Archivematica itself will get installed on the same machine as the storage service.  Set gid is probably not a good idea for the storage service, I think?  Only the one user (archivematica) should be able to write to the storage service logs..  For Archivematica the situation is a little different because of the archivematicadashboard user.  The storage service would prefer g+rs I think?  

Maybe the storage service should be logging to /var/log/archivematica/storage-service, and the dashboard to /var/log/archivematica/dashboard?  We were basing this layout on things like nginx and elasticsearch, which do not use setgid.

mistydemeo replied:
That sounds like a good idea, actually - I like the idea of separating the log paths for the different components into subdirectories; better than setting gid.

Hwesta commented:
Looks good to me.

jhsimpson committed:
Moving default logging location

as mentioned in PR63, move the default logging location
for the storage service to:

/var/log/archivematica/storage-service/

This way other apps in the archivematica ecosystem can log to
their own subdirectories in /var/log/archivematica/ as well
and each subdir can have its own ownership or permissions if required.

jhsimpson commented:
I changed the default location of storage service logs in efc6e99.  I will merge this now.
